ast/src/parsed/mod.rs

```diff
@@ -151,7 +152,7 @@ impl<T> PilStatement<T> {
     }
 }
 
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
```

Comment:
Are you sure the structs in this file also need serde? I'd imagine only `analyzed/*` would need it for `Analyzed<T>`, but also not sure right now

Reply:
Because the `Analyzed<T>` uses some of those directly or indirectly, we definitely have to make them serializable yes :)
For example, the `definitions` field of that struct contains a `FunctionValueDefinition<T>` that itself contains a `Expression<T>` in the `parsed/*` folder (and thus also all of its children)